src/main/java/uk/gov/hmcts/reform/hmc/service/LinkedHearingGroupServiceImpl.java

```diff
@@ -172,16 +187,45 @@ private List<HearingResponseEntity> getLatestVersionHearingResponses(HearingEnti
 
     private void deleteFromLinkedGroupDetails(List<HearingEntity> linkedGroupHearings) {
         LinkedGroupDetails linkedGroupDetails = linkedGroupHearings.get(0).getLinkedGroupDetails();
+        final String requestId = linkedGroupDetails.getRequestId();
         saveLinkedGroupDetailsAudit(linkedGroupDetails);
         linkedGroupHearings.forEach(hearingEntity -> saveLinkedHearingDetailsAudit(hearingEntity));
-        saveLinkedGroupDetails(linkedGroupDetails);
-        // TODO: call ListAssist - https://tools.hmcts.net/jira/browse/HMAN-97
+        saveLinkedGroupDetails(linkedGroupDetails, requestId);
+        try {
+            futureHearingRepository.deleteLinkedHearingGroup(requestId);
+            log.info("Response received from ListAssist successfully");
+            linkedGroupDetailsRepository.delete(linkedGroupDetails);
+        } catch (Exception exception) {
+            processResponseFromListAssistForDeleteLinkedHearing(linkedGroupDetails, exception);
+        }
```

Comment:
```suggestion
        } catch (BadFutureHearingRequestException exception) {
            process400ResponseFromListAssistForDeleteLinkedHearing(linkedGroupDetails, exception);
        } catch (AuthenticationException exception) {
            process500ResponseFromListAssistForDeleteLinkedHearing(...);
        }
```
Rather than having to check what type of error it is in the method that currently processes these. Probably makes it a bit easier to read the code too? The current code in `processResponseFromListAssistForDeleteLinkedHearing()` will also fail with casting in the `else` whenever it's another exception that isn't an `AuthenticationException` (e.g. database issue), but this will prevent that too.